collect_app/src/main/java/org/odk/collect/android/widgets/SelectWidget.java

```diff
@@ -46,15 +46,6 @@ public SelectWidget(Context context, FormEntryPrompt prompt) {
         answerLayout = new LinearLayout(context);
         answerLayout.setOrientation(LinearLayout.VERTICAL);
         playList = new ArrayList<>();
-
```

Comment:
There should be a call on `readItems()` here, right? Otherwise I believe this breaks the other way to do dynamic csv choices - http://xlsform.org/#dynamic-selects-from-pre-loaded-data

@mmarciniak90 Do you have test cases related to this `search()` appearance/function? I don't use it so I tend to forget about it but a lot of users do rely on it.

Reply:
I'll check that form again but I'm pretty sure it's ok since it's called here https://github.com/opendatakit/collect/pull/1969/files#diff-757ac06637936c20634d6e7b1d560ecdR61 so there is no need to do that in the abstract method.

Reply:
👍 You're right, thank you. Pulling out `readItems` and calling `createLayout` in the constructors would have been a good commit to separate out. Now I can see you made that change in several stages.

Reply:
I confirmed that form works well.

Reply:
@lognaturel yes, I verified all `select one` widgets available in `All widgets` form, also with search.